test-configs.yaml

```diff
@@ -831,35 +815,53 @@ device_types:
       - whitelist: {defconfig: ['defconfig']}
       - blacklist: {kernel: ['v3.']}
 
-  qemu_arm:
+  qemu_i386:
```

Comment:
Please keep the entries in alphabetical order.

Reply:
ok